pkgs/top-level/python-packages.nix

```diff
@@ -7368,6 +7370,8 @@ self: super: with self; {
 
   openaiauth = callPackage ../development/python-modules/openaiauth { };
 
+  openapi-codec = callPackage ../development/python-modules/openapi-codec { };
```

Comment:
This change should belong into the other commit

Reply:
Fixed.